dataprep/connector/assets/info.txt

```diff
@@ -0,0 +1,117 @@
+<html>
```

Comment:
Renaming it to info.html will be better (and you can get syntax highlight in the editor)